price-feeder/config/config.go

```diff
@@ -200,6 +201,17 @@ func ParseConfig(configPath string) (Config, error) {
 		}
 	}
 
+	for quote := range coinQuotes {
```

Comment:
maybe add a comment to clarify why we use the  `coinQuotes`
`// coinQuotes checks whether a currency pair doesn't have a quote in USD or it has some base quoted in USD`

Reply:
ada85eead28583ff582c1a17dd54a591e6cd2966